modules/eslint-plugin/spec/rules/no-dispatch-in-effects.spec.ts

```diff
@@ -51,6 +51,42 @@ class Ok2 {
       { dispatch: false },
     )
   }
+}`,
+  `
```

Comment:
Could you split these test cases up into two sections please?
One for the cases using a constructor, the others for the ones using `inject`.
As an example see, https://github.com/ngrx/platform/pull/3951/commits/8c8f4feff57ed809de6a18edf0b5b47364a488f1
This way I find it more more clear.

This comment is for all the open PRs.